test/benchmark_tests/001-atlas/main

```diff
@@ -0,0 +1,26 @@
+
+cvmfs_test_name="ATLAS benchmark"
+cvmfs_test_autofs_on_startup=false
+cvmfs_benchmark="yes"
+
+FQRN=atlas.cern.ch
+
+
+cvmfs_run_benchmark() {
+  export ATLAS_LOCAL_ROOT_BASE=/cvmfs/atlas.cern.ch/repo/ATLASLocalRootBase
```

Comment:
It's necessary to define that variable inside the test, otherwise it doesn't run properly. Concerning the pollution, for the warm cache creation it runs on purporse like `bash -c "cvmfs_run_benchmark"`, and during the normal run it's also the same by default. This way it doesn't propagate the value of the variable.


Just in case I've just tried it out with the following script:

bash -c 'export OH_GOD="ohmygod"'
echo $OH_GOD

... and it prints nothing.